lib/peer_manager/Kconfig

```diff
@@ -40,7 +40,7 @@ config PM_FLASH_BUFFERS
 	  Number of internal buffers for flash operations.
 	  Decrease this value to lower RAM usage.
 
-config PM_SERVICE_CHANGED_ENABLED
+config PM_SERVICE_CHANGED
 	bool "Enable/disable the service changed management for GATT server in Peer Manager."
```

Comment:
```suggestion
	bool "Service changed management for GATT server in Peer Manager."
```

Reply:
Please have another look, this is addressed in a separate commit.